packages/react-charting/config/constants.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
>module [](http://example.com/codeflow?start=0&length=6)

move this file inside react-charting/scripts

Reply:
done